### PR TITLE
[New] adding the keras-nlp and hf integration blog post

### PR DESCRIPTION
**Pull request by ariG23498**
This PR will add the blog post about the Keras NLP and HF integration.

CC: @gante

### CODE REVIEW

**Review by gante — COMMENTED** [review]
Very nice!

Missing:
- Updating the `_blog.yaml` file, containing this blog post entry (see [here](https://github.com/huggingface/blog/pull/2176/files#diff-a332b83464cf2b650715bacb6e3f07b994af0790acc88a4ea353883ba2ae751e) for an example)

Double-check:
- Have you previewed the blog post for visual nits? Instructions [here](https://github.com/huggingface/blog?tab=readme-ov-file#more-assistance)

File: keras-nlp-integration.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+---
+title: "Announcing New Hugging Face and Keras NLP integration" 
+thumbnail: /blog/assets/todo
```
[the remark this answers is not in the document]
Comment (ariG23498) [reply keras-nlp-integration.md]: Thanks for the pointers!

**Comment by ariG23498** [comment]
@gante does the thumbnail look okay? This was my first time building an asset for the blog post 😅 

Also I have double checked the rendering -- seems okay to me.

File: _blog.yml [hunk 2]
```diff
@@ -4319,3 +4319,11 @@
...
     - inference
     - hardware
+
+- local: keras-nlp-integration
+  title: "Announcing New Hugging Face and Keras NLP integration"
+  thumbnail: /blog/assets/keras-nlp-integration/thumbnail.png
+  author: ariG23498
+  date: Jul 9, 2024
+  tags:
+    - keras-nlp
```
Comment (gante) [inline _blog.yml:4329]: Tags has to be one or more items of these: https://github.com/huggingface/blog/blob/main/_tags.yml

**Review by gante — COMMENTED** [review]
A final nit!

**Review by gante — APPROVED** [review]
LGTM 👍

**Pull request merged**